core/backend/src/test/imodel/IModel.test.ts

```diff
@@ -1685,6 +1685,20 @@ describe("iModel", () => {
   });
 
   it("should be able to reproject with iModel coordinates to or from any other GeographicCRS", async () => {
+    const addGcsWs = async (id: string) => {
+      try {
+        const ws = await IModelHost.appWorkspace.getContainer({ id });
+        const fileName = ws.localFile;
+        IModelHost.appWorkspace.dropContainer(ws);
+        expect(IModelHost.platform.addGcsWorkspace(fileName)).to.be.true;
+      } catch (e) {
+        // eslint-disable-next-line no-console
+        console.log(`cannot load GCS Workspace: ${id}`);
+      }
+    };
+
+    await addGcsWs("usa");
```

Comment:
I will set the final name of the workspace once it is decided. The tests currently fall back on normal grid files when workspace is absent. I suppose once the upack is ready the build process will copy the workspace files in %localappdata%/itwinjs/Worspaces which raises the question of workspace versions. Although the workspaces as a set are compatible and the set should be useable by older software versions, we need to insure the newer version is retained when software install is performed. Do we version workspaces or do we relegate all this to software install process (a GCSversion.txt file in the Workspace directory? or a sub-directory with a versioned name just like UPacks?)

Reply:
The workspace files should come from cloud storage when we get that set up, not from the build process. `IModelHost.appWorkspace.getContainer` will download the WorkspaceFile if necessary. 

Workspaces are versioned, although when will we ever have a new version of the gcs data and why?

Reply:
The BaseGCS workspace containing the *.dty files will get updated about every 1 or 2 months but as for the others they rarely are modified and only to add a new grid file. If the cloud container has always the latest version then it will be fine.